Toolset/libraries/revfrontscriptlibrary.livecodescript

```diff
@@ -287,20 +289,6 @@ on openStack
       end if
    end if
 
-   # OK-2008-03-19 : Modifications for new debugger
-   # Activate breakpoints for newly opened stack, but only if its a real "openStack" as opposed
-   # to something doing "send openstack to button revfrontscript"
-
-   local tTarget
-   put the long id of the target into tTarget
-   if tTarget is not the long id of me then
-      local tStack
-      put revTargetStack(tTarget) into tStack
-      if revDebuggerEnabled() then
-         revDebuggerActivateBreakpoints the long id of stack tStack
-      end if
-   end if
-
```

Comment:
@montegoulding These changes don't seem to be related to the `_ideoverride` propery, as far as I can tell?